doc/03_reference/verification.rst

```diff
@@ -89,20 +89,16 @@ Prerequisites & Environment Setup
 
 In order to run the co-simulation flow, you'll need:
 
-- A SystemVerilog simulator that supports UVM. The flow is currently
-  tested with VCS.
+- A SystemVerilog simulator that supports UVM. The flow is currently tested with VCS.
```

Comment:
Nit: I suppose these two sentences should be on separate lines, to match the usual style

Reply:
Well spotted, guess I accidentally reflowed the text.

Sadly the one line, one sentence style isn't repeated throughout the file nor is it wrapped to 80 characters where this style is broken. I'll open an issue to note this should get sorted.